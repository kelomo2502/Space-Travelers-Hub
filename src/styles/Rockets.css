.Rockets {
  padding-bottom: 88px;
}
