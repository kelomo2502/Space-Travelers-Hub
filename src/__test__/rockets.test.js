import React from 'react';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import store from '../redux/configurestore';
import Rockets from '../components/rockets';

it('Renders Rockets component correctly', () => {
  const tree = renderer.create(
    <Provider store={store}>
      <Rockets />
    </Provider>,
  );
  expect(tree).toMatchSnapshot();
});
