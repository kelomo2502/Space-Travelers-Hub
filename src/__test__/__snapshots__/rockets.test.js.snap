// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Rockets component correctly 1`] = `
<div
  className="Rockets"
/>
`;
